lexicons/tools/ozone/history/defs.json

```
@@ -0,0 +1,166 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.history.defs",
+  "defs": {
+    "subjectBasicView": {
+      "type": "object",
+      "required": ["subject", "status", "modAction"],
+      "properties": {
+        "subject": {
+          "type": "string",
+          "format": "uri"
+        },
+        "status": {
+          "type": "string"
+        },
+        "modAction": {
+          "type": "string",
+          "knownValues": [
+            "#modActionPending",
+            "#modActionLabel",
+            "#modActionResolve",
+            "#modActionSuspend",
+            "#modActionTakedown"
+          ]
+        },
+        "subjectProfile": {
+          "type": "ref",
+          "ref": "app.bsky.actor.defs#profileViewBasic"
+        },
+        "labels": {
+          "type": "array",
+          "items": {
+            "type": "ref",
+            "ref": "com.atproto.label.defs#label"
+          }
+        }
+      }
+    },
+    "eventView": {
+      "type": "object",
+      "required": ["subject", "createdAt", "event", "isAutomated"],
+      "properties": {
+        "subject": {
+          "type": "string",
+          "format": "uri"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "isAutomated": {
+          "type": "boolean",
+          "default": false
```

Since this is required the default wont come into play.
```suggestion
          "type": "boolean"
```